localization/sparse_mapping/tools/merge_bags.py

```diff
@@ -1,77 +0,0 @@
-#!/usr/bin/env python
-# Copyright (c) 2017, United States Government, as represented by the
-# Administrator of the National Aeronautics and Space Administration.
-# 
-# All rights reserved.
-# 
-# The Astrobee platform is licensed under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with the
-# License. You may obtain a copy of the License at
-# 
-#     http://www.apache.org/licenses/LICENSE-2.0
-# 
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-
-# Merge bags.
-# Usage:
-# python merge_bags.py output.bag input*bag
-
-import sys
-import argparse
-from fnmatch import fnmatchcase
-
-from rosbag import Bag
-
-def main():
-
-    parser = argparse.ArgumentParser(description='Merge one or more bag files with the possibilities of filtering topics.')
-    parser.add_argument('outputbag', help='Output bag file with topics merged.')
-    parser.add_argument('inputbag', nargs='+', help='Input bag files.')
-    parser.add_argument('-v', '--verbose', action="store_true", default=False,
-                        help='Verbose output.')
-    parser.add_argument('-t', '--topics', default="*",
```

Comment:
The previous version had an option to filter topics, is this something we still need in the merge bags script? @rsoussan @marinagmoreira @kbrowne15

Reply:
I don't need it. We can filter out topics after the bags are merged using the rosbag tool.